pyworkflow/pyworkflow/workflow.py

```diff
@@ -13,9 +13,11 @@ class Workflow:
         file_path: Location of a workflow file
     """
 
-    def __init__(self, graph=nx.DiGraph(), file_path=None):
+    def __init__(self, graph=nx.DiGraph(), file_path=None, workflow_name='a-name'):
+        #TODO: need to discuss a way to generating the workflow name. For now passing a default name.
```

Comment:
We can use the diagram UUID when hitting the `workflow/new` endpoint. Either in a querystring param, or by making the endpoint POST

Reply:
@reddigari that sounds like a good idea. I'll go ahead with that.